tests/robottelo/test_vm.py

```diff
@@ -14,9 +14,10 @@ class VirtualMachineTestCase(unittest.TestCase):
     def setUp(self):
         super(VirtualMachineTestCase, self).setUp()
         self.properties_backup = conf.properties.copy()
+        conf.properties['clients.provisioning_server'] = None
```

Comment:
All our tests are doing `self.configure_provisoning_server()` to set provisioning server, except the test, mentioned before - its purpose is to verify that without setting the server assertion will be risen.